Userland/seq.cpp

```diff
@@ -56,6 +67,13 @@ int main(int argc, const char* argv[])
         return 1;
     }
 
+    for (int i = 1; i < argc; i++) {
+        if (strcmp(argv[i], "--help") == 0 || strcmp(argv[i], "-h") == 0) {
+            print_usage(stdout);
+            exit(0);
+        }
+    }
+
```

Comment:
We have `Core::ArgsParser` for this :)

Reply:
Unfortunately, `ArgsParser` cannot really deal with the...odd argument format here (yet?)

Reply:
I somehow thought one could give it a custom help text but I confused that with explicit calls of `args_parser.print_usage()` (which still shows the generated help text). Nevermind!